### PR TITLE
PCHR-1576: Menu updates

### PR DESCRIPTION
**Pull request by deb1990**
# Requirement
1. On mobile version remove a separate dropdown for logo. 
   image
2. On click on logo let's open the menu under the burger icon with those two links moved to the first dropdown
# Fix
1. Removed the dropdown shown on above screenshot.
2. Added 2 new menu items which is shown only on mobile version(using CSS).
   anim
3. Open menu on clicking the logo

Note: Adding the menu is taken care by 
https://github.com/compucorp/civihr-employee-portal/pull/201

### CODE REVIEW

**Comment by ghost** [comment]
Looks good to me

**Comment by mi-kon** [comment]
LGTM

**Comment by AkA84** [comment]
Is it normal that those two links appear just like the other ones? Seems to me they should be visually separated or highlighted from the rest

**Comment by mi-kon** [comment]
Yeah, I think it's fine for now.

**Review by AkA84 — APPROVED** [review]

**Comment by ghost** [comment]
Reposted from https://github.com/compucorp/civihr-employee-portal/pull/201

> I think it would be worth differentiating the Admin and SSP links from the others as they serve a different purpose and so should sit differently in the hierarchy. Is this something that's possible to do?

**Comment by deb1990** [comment]
@JonScreat What exactly should we do.  A different border/background or something else?

**Comment by ghost** [comment]
I suggest something like this, using the grey background to show the importance of these two options.

ssp menu m - alt 2x

**Comment by deb1990** [comment]
Menu padding fixed 
image

**Comment by ghost** [comment]
Looks good to me

**Review by AkA84 — APPROVED** [review]

**Comment by deb1990** [comment]
SSP bottom border removed
image

File: civihr_default_theme/scss/partials/_menu.scss [hunk 1]
```diff
@@ -131,6 +131,10 @@
     background-color: $gray-light;
   }
 
+  .menu-link-civihr-ssp>a {
```
Comment (AkA84) [inline civihr_default_theme/scss/partials/_menu.scss]:
``` scss
.menu-link-civihr-ssp > a {

}
```

please leave a space around the `>`

(applies for all other instances)

**Review by AkA84 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Looks good to me

**Review by AkA84 — APPROVED** [review]

**Pull request merged**